benchmark/src/main/resources/logback.xml

```diff
@@ -0,0 +1,27 @@
+<!--
+  ~ Copyright 2020 LINE Corporation
+  ~
+  ~ LINE Corporation licenses this file to you under the Apache License,
+  ~ version 2.0 (the "License"); you may not use this file except in compliance
+  ~ with the License. You may obtain a copy of the License at:
+  ~
+  ~   https://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  ~ WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  ~ License for the specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+<configuration>
```

Comment:
- logback is complaining that there are two `logback.xml` in classpath... Could you handle this?
- And could you suppress these logback's INFO logs?

```
10:43:14,766 |-INFO in ch.qos.logback.classic.LoggerContext[default] - Could NOT find resource [logback.groovy]
10:43:14,767 |-INFO in ch.qos.logback.classic.LoggerContext[default] - Found resource [logback.xml] at [jar:file:/Users/JP22047/develop/src/github.com/line/decaton/benchmark/build/libs/benchmark-0.0.36-SNAPSHOT.jar!/logback.xml]
10:43:14,768 |-WARN in ch.qos.logback.classic.LoggerContext[default] - Resource [logback.xml] occurs multiple times on the classpath.
10:43:14,768 |-WARN in ch.qos.logback.classic.LoggerContext[default] - Resource [logback.xml] occurs at [jar:file:/Users/JP22047/develop/src/github.com/line/decaton/testing/build/libs/testing-0.0.36-SNAPSHOT.jar!/logback.xml]
10:43:14,768 |-WARN in ch.qos.logback.classic.LoggerContext[default] - Resource [logback.xml] occurs at [jar:file:/Users/JP22047/develop/src/github.com/line/decaton/benchmark/build/libs/benchmark-0.0.36-SNAPSHOT.jar!/logback.xml]

```

Reply:
ok, so moving logback.xml for `testing` module into `processor/it`?

Reply:
+1